src/Cake.Common/Tools/TextTransform/TextTransformAliases.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.TextTransform
+{
+    /// <summary>
+    /// Contains functionality related to transforming templates
+    /// </summary>
+    [CakeAliasCategory("TextTransform")]
+    public static class TextTransformAliases
+    {
+        /// <summary>
+        /// Transform a text template.
+        /// </summary>
+        /// <example>
+        /// <code>
+        /// // Transform a .tt template.
+        /// var transform = File("./src/Cake/Transform.tt")
+        /// TransformTemplate(transform, new TextTransformSettings { OutputFile="./src/Cake/Transform.cs" });
```

Comment:
@devlead @patriksvensson If you don't provide an output file name, will the T4 template still run?  If not, then OutputFile should be moved from TextTransformSettings to a parameter of the Alias.